services_and_applications/slate/overview.rst

```diff
@@ -0,0 +1,37 @@
+*****************************
+Overview
+*****************************
+
+*Slate* is the high-level service name referring to two user
+facing OpenShift clusters.
```

Comment:
```suggestion
*Slate* provides container orchestration services that support a user project's existing Summit and Alpine allocations. The Slate service today consists two user facing OpenShift clusters which each provide capabilities unique to the security enclave that they are in.
```

Reply:
I don't love the wording but I think we should talk more about what Slate is overall